src/index.tsx

```diff
@@ -452,6 +465,60 @@ export class App extends React.Component<{}, AppState> {
               </>
             )}
 
+            {hasText(elements) && (
+              <>
+                <h5>Font size</h5>
+                <ButtonSelect
+                  options={[
+                    { value: 16, text: "16 px" },
```

Comment:
Could you pick 4 and name them "Small", "Medium", "Large", "Very Large" or something. I'd rather have fewer options if possible

Reply:
Great idea !!! it's done